package/clp.py

```diff
@@ -0,0 +1,44 @@
+import urllib.request
+import re
+
+def grab(url_name):
```

Comment:
use function annotations for better reabability, since it uses new feature. we can explicitly add them(just in case)

```suggestion
def grap(url_name: str) -> str:
```